MLonCitrination.ipynb

```diff
@@ -377,15 +372,17 @@
      "name": "stdout",
      "output_type": "stream",
      "text": [
-      "We predict the density of AlCu to be 7.46168347193 +/- 2.19360592891\n"
+      "1 1\n",
+      "We predict the density of AlCu to be 7.464366567871429 +/- 2.178253940554062\n"
      ]
     }
    ],
    "source": [
     "inputs = [{\"Chemical formula\": \"AlCu\"},]\n",
-    "resp = client.predict(\"27\", inputs)\n",
-    "prediction = resp['candidates'][0]['Density']\n",
-    "print(\"We predict the density of {} to be {} +/- {}\".format(inputs[0]['Chemical formula'], *prediction))"
+    "resp = client.models.predict(\"27\", inputs)\n",
+    "print(len(inputs), len(resp))\n",
+    "prediction = resp[0].get_value('Density')\n",
+    "print(\"We predict the density of {} to be {} +/- {}\".format(inputs[0]['Chemical formula'], prediction.value, prediction.loss))"
```

Comment:
Not a big deal, but do we want to be consistent with design table results here?

`round(prediction.value, 2)`